float64 sonar_front
float64 sonar_behind
float64 sonar_left
float64 sonar_right
float64 sonar_up
float64 sonar_down
float64 sonar_cam
